docs/docs/getting-started.server.mdx

```diff
@@ -842,6 +870,8 @@ See their readmes on how to configure them.
 *   If you’re getting errors integrating MDX, see
     [§ Troubleshooting MDX][trouble] or [§ Support][support]
 
+[@next/mdx]: https://github.com/vercel/next.js/tree/canary/packages/next-mdx
```

Comment:
Should we link to the Canary branch? Is that the default branch?

Reply:
Yep, it’s their default branch.